schema/v2/cards_schema.json

```diff
@@ -265,6 +265,17 @@
         }
       ]
     },
+    "layout_id": {
+      "oneOf": [
+        {
+          "type": "string",
+          "enum": ["normal", "flip", "copy", "facade", "progression"]
```

Comment:
single_sided instead of normal?  Not convinced it's better, but wdyt?

Reply:
I have no preference. On a technicality not all non-normal layouts are multi-sided (copy cards are distinct single-sided cards, and the only printed progression cards use stickers). I'm not going to argue for either though; I'll make it whatever you think is best.

Reply:
I think "single_sided" is more descriptive than "normal".

Reply:
I'm happy to concede to majority opinion.